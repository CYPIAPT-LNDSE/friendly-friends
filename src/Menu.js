import React, { Component } from 'react';
import Button from './Button.js';

class Menu extends Component {
  render() {
    return (
      <header className="menu">
        <Button label="stroke" />
        <Button label="talk" />
        <Button label="feed" />
        <Button label="play" />
      </header>
    );
  }
}

export default Menu;
