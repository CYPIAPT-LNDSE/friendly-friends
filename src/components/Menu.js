import React from 'react';
import Button from './Button.js';

const Menu = (props) => {
  return (
    <header className="menu">
      <Button label="stroke" />
      <Button {...props} label="talk" />
      <Button label="feed" />
      <Button label="play" />
    </header>
  );
}

export default Menu;
