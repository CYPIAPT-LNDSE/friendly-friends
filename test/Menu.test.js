import React from 'react';
import renderer from 'react-test-renderer';
import Menu from '../src/components/Menu.js';

test('Menu renders correctly', () => {
  const tree = renderer.create(<Menu/>).toJSON();
  expect(tree).toMatchSnapshot();
});
