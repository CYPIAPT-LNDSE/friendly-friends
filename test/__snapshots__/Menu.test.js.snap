exports[`test Menu renders correctly 1`] = `
<header
  className="menu">
  <button
    className="button">
    stroke
  </button>
  <button
    className="button">
    talk
  </button>
  <button
    className="button">
    feed
  </button>
  <button
    className="button">
    play
  </button>
</header>
`;
